Mathlib/Algebra/Category/ModuleCat/Differentials/Presheaf.lean

```diff
@@ -81,7 +81,8 @@ def postcomp (f : M ⟶ N) : N.Derivation φ where
   d_map {X Y} g x := by simpa using naturality_apply f g (d.d x)
   d_app {X} a := by
     dsimp
-    erw [d_app, map_zero]
+    erw [d_app]
+    rw [map_zero]
```

Comment:
These splits are intentional, so it's easier to use `erw?`, and generally to make clearer where the problem is.